system-int-test/test/cmr/system_int_test/ingest/subscription_time_frame_job_test.clj

```diff
@@ -0,0 +1,60 @@
+(ns cmr.system-int-test.ingest.subscription-time-frame-job-test
+  "CMR subscription time-frame job tests."
+  (:require
+   [cheshire.core :as json]
+   [clj-http.client :as client]
+   [clojure.test :refer [deftest is join-fixtures testing use-fixtures]]
+   [cmr.common.util :as util :refer [are3]]
+   [cmr.mock-echo.client.echo-util :as echo-util]
+   [cmr.system-int-test.system :as system]
+   [cmr.system-int-test.utils.dev-system-util :as dev-system]
+   [cmr.system-int-test.utils.ingest-util :as ingest]
+   [cmr.system-int-test.utils.url-helper :as url]))
+
+(use-fixtures :each
+  (join-fixtures
+   [(ingest/reset-fixture {"provguid1" "PROV1" "provguid2" "PROV2"}
+                          {:grant-all-search? false
+                           :grant-all-ingest? false})
+    (dev-system/freeze-resume-time-fixture)]))
+
+(defn- revision-date-range-query-with-response-status
+  [query-params expected-status]
+  (let [response (client/post (url/email-subscription-processing)
+                              {:accept "application/json"
+                               :query-params query-params
+                               :connection-manager (system/conn-mgr)
+                               :throw-exceptions false})]
+    (is (= expected-status
+           (:status response)))))
+
+(deftest subscriptions-job-endpoint-test
+  (let [admin-update-group (echo-util/get-or-create-group (system/context) "admin-update-group")
```

Comment:
nit: I would separate this test into two test cases (i.e. `(testing ...`). One for permission, and one for revision-date-range validation. This will make the test cleaner.

Reply:
Actually I added the endpoint to `cmr.system-int-test.admin.admin-permissions-test` so I think it's safe enough for me to just drop the permissions tests since they should be covering duplicate functionality.

Reply:
Since you are not doing any of the permission test now, you can remove the permission setup in fixture and the admin token stuff from your test and perhaps update the namespace and test name to indicate that you are only testing revision-date-range validation.

Reply:
It would be nicer to test the error message as well, e.g. this is one of your test case:
```
"returns 400 when only a start-date is provided"
"2000-01-01T10:00:00Z,"
400
"Missing end date in revision-date-range."
```